.github/workflows/ci.yml

```diff
@@ -28,6 +33,14 @@ jobs:
       - name: build
         run: pnpm build
 
+      - name: upload-dist
+        uses: actions/upload-artifact@v4
+        if: github.event_name == 'release'
+        with:
+          name: dist
+          path: dist/
+          retention-days: 1
```

Comment:
Is there any reason to choose 1 day for artifact retention time?

Reply:
we don't need it once it is added to release assets

Reply:
this artifact and release assets are different things